.circleci/config.yml

```diff
@@ -19,9 +19,9 @@ jobs:
       - checkout
       - login
       - run: yarn install --frozen-lockfile
-      - run: yarn build
-      - run: yarn lint
-      - run: yarn test
+      - run: yarn ci:test
+      - store_test_results:
+          path: ./reports/unit_tests/
```

Comment:
Is there a specific reason why you are storing the test results ? We're not doing coverage checks on `activeui-migration`.

Reply:
I thought this was needed to see reports of failing tests on circleci. I will check how circleci behaves without it.

Reply:
Ah you're probably right, I'll let you double check

Reply:
I've checked and although it does not show anything anymore under the "tests" tab, maybe the console output is enough?